src/components/Profile/ProfileLink/ProfileLink.css

```diff
@@ -3,13 +3,16 @@
   align-items: center;
   gap: 0.5em;
   padding: 0.125em;
-
-  span {
-    font-size: 14px;
-  }
 }
 
 .profile__header__link a {
+  display: flex;
+  align-items: center;
   text-decoration: none;
   color: inherit;
 }
+
+.profile__header__link a span {
+  font-size: 14px;
+  margin-left: 0.5em; 
```

Comment:
We are using `gap` in the project, margin is usually not recommended in modern CSS ([see this](https://youtu.be/KVQMoEFUee8?si=kVjm_vw8YK0jOeXM)).